t/after.t

```diff
@@ -0,0 +1,20 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Path::Tiny;
+use Rex::Commands;
+
+$::QUIET = 1;
+
+my $tmpfile = Path::Tiny->tempfile;
```

Comment:
Do we absolutely need to introduce a new non-core dependency for working with a tempfile? Other tests use `File::Temp` or `Rex::Helper::Path::get_tmp_file()` - can one of them used here too?